lib/processes.js

```diff
@@ -50,7 +50,7 @@ function launchAllProcesses(config) {
   }
 
   // app.port is special because the proxy needs to know it
-  var appPort = launchApp ? config.get('app.port', 0) : -1;
+  var appPort = config.get('app.port', launchApp ? 0 : 41998);
```

Comment:
Actually, this could be -1 as well. Misread an existing `.testiumrc` of someone using `launch=false` and thought we used to have that as a default port.